docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+
+Monitoring with StatsD
+::::::::::::::::::::::
+
+Requires the ``statsd`` package, or *Cliquet* installed with
+``pip install cliquet[monitoring]``.
+
+StatsD metrics can be enabled (disabled by default):
+
+.. code-block:: ini
+
+    cliquet.statsd_url = udp://localhost:8125
+    # cliquet.statsd_prefix = cliquet.project_name
+
+
+Monitoring with New Relic
+:::::::::::::::::::::::::
+
+Requires the ``newrelic`` package, or *Cliquet* installed with
+``pip install cliquet[monitoring]``.
+
+Enable middlewares as described :ref:`here <configuration-middlewares>`.
+
+New-Relic can be enabled (disabled by default):
+
+.. code-block:: ini
+
+    cliquet.newrelic_config = /location/of/newrelic.ini
+    cliquet.newrelic_env = prod
+
+.. _configuration-authentication:
+
+Authentication
+==============
+
+By default, *Kinto* relies on *Basic Auth* to authenticate users.
+
+User registration is not necessary. A unique user idenfier will be created
+for each couple of ``username:password``.
+
+*Kinto* is compatible with *Firefox Account*.  Install and
+configure :github:`mozilla-services/cliquet-fxa`.
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.userid_hmac_secret ``''``     | The secret used by the server to derive the shareable user id. This      |
+|                                       | value should be unique to each instance and, of course, kept secret. By  |
+|                                       | default, Kinto doesn't define a secret for you, and won't start unless   |
+|                                       | you generate one.                                                        |
++---------------------------------------+--------------------------------------------------------------------------+
+| multiauth.policies ``["basicauth",    | `MultiAuthenticationPolicy <https://github.com/mozilla-                  |
+| ]``                                   | services/pyramid_multiauth>`_ is a Pyramid authentication policy that    |
+|                                       | proxies to a stack of other IAuthenticationPolicy objects, to provide a  |
+|                                       | combined auth solution from individual pieces. Simply pass it a list of  |
+|                                       | policies that should be tried in order.                                  |
++---------------------------------------+--------------------------------------------------------------------------+
+| multiauth.policy.basicauth.use        | Python *dotted* path to the authentication policy to use for basicauth.  |
+| ``('cliquet.authentication.           | By default, every couple of login / password will be accepted, meaning   |
```

Comment:
As above:
> By default, every login / password pair will be [...]

That said, do you mean "every" or "any" here? Every implies that multiple pairs could be passed in a single request, whereas "any" means that literally any pair will be accepted (but only one per request).